src/App.jsx

```diff
@@ -135,71 +136,62 @@ function App() {
 
 	// RENDER TODOS
 	return (
-		<div className='wrapper'>
-			<div className='AppContainer'>
-				<ToastContainer
-					position='top-right'
-					autoClose={5000}
-					hideProgressBar={true}
-					newestOnTop={false}
-					closeOnClick
-					rtl={false}
-					pauseOnFocusLoss
-					draggable
-					pauseOnHover
-					theme='dark'
-				/>
-				<BrowserRouter>
-					<Routes>
-						<Route
-							exact
-							path='/'
-							element={
-								<>
-									<Link to='/new'>
-										<MdOutlinePlaylistAddCheck className='btn-completed' />
-									</Link>
-									<AddTodoForm
-										todoListName={'TODOS'}
-										numberTodos={todoList.length}
-										onAddTodo={addTodo}
-										isMuted={isMuted}
-										setIsMuted={setIsMuted}
-										loadTodos={loadTodos}
-									/>
-									{isLoading ? (
-										<p>Loading...</p>
-									) : (
-										<TodoList
-											todoList={todoList}
-											onUpdateTodo={updateTodo}
-											onRemoveTodo={removeTodo}
+		<>
+			<ConfiguredToast/>
```

Comment:
Moved toast higher in the element tree to minimize the chance of inherited styles interfering with the toast's own styling.